Backend/src/main/java/de/baleipzig/pdfextraction/backend/workunits/XDocWU.java

```diff
@@ -83,4 +88,19 @@ private void checkResult(final Map<Field, String> result) {
 
         return processed.toByteArray();
     }
+
+    private String parseSenderName(String senderAddress){
+
+        return senderAddress.split("\n")[1];
+    }
+
+    private String convertSenderToReceiver(String sender){
+
+        return sender.replace(", ", "\n");
+    }
+
+    private String convertReceiverToSender(String receiver){
+
+        return receiver.replace("\n", ", ");
+    }
```

Comment:
Warum ist das hier drinnen?

Reply:
Da hab ich scheinbar nen Fehler beim committen oder so gemacht, das gehörte ja eigentlich in einen anderen Branch